src/components/Heading/Heading.module.css

```diff
@@ -0,0 +1,67 @@
+/* sizes */
+
+.heading {
+  @apply mt-0 mb-0;
+}
```

Comment:
with size being required, don't need to set text-body and font-normal, should always be bold weight